apps/web/src/components/display/AddRecordModal.tsx

```diff
@@ -100,21 +102,34 @@ export const AddRecordModal: React.FC<AddRecordProps> = (props: AddRecordProps)
             <FormikForm>
               <div className='grid grid-cols-4 gap-4 bg-white rounded px-8 pt-6 pb-7'>
                 <div className='mb-3 col-span-2'>
-                  <Field
-                    name='ha_pcn'
-                    label='Health Authority'
-                    component={({ field, form }: FieldProps) => (
-                      <ReactSelect<RecordTypeOptions>
-                        inputId={field.name}
-                        value={haPcn?.data?.find(s => s.id == field.value)}
-                        onBlur={field.onBlur}
-                        onChange={value => form.setFieldValue(field.name, `${value?.id}`)}
-                        options={haPcn?.data?.map(s => ({ ...s, isDisabled: s.id == field.value }))}
-                        getOptionLabel={option => `${option.title}`}
-                        styles={getSelectStyleOverride<RecordTypeOptions>()}
-                      />
-                    )}
-                  />
+                  {authUser?.ha_pcn_id ? (
+                    <Field
```

Comment:
How about adding `isDisabled` prop to Select component?

Reply:
main reason I added a Field instead of reusing the Select is the longer HA's get cut off in the select, but a full text Field shows them